astropy/coordinates/representation.py

```diff
@@ -219,6 +219,21 @@ def to_cartesian(self):
         # Note: the above docstring gets overridden for differentials.
         raise NotImplementedError()
 
+    def to_cartesian_with_differentials(self):
```

Comment:
@mhvk @eteq I could use this for brevity in the affine transform, so this is one possibility for a short-hand